.github/workflows/code-review.yaml

```diff
@@ -0,0 +1,54 @@
+# This workflow will perform the code review on each code change.
+# It uses "actions/cache" and turborepo in order to perform tasks only on changed files.
+
+name: "Code Review"
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened, ready_for_review]
+    paths:
+      - apps/**
+      - packages/**
+      - package.json
+  workflow_dispatch:
+
+jobs:
+  code_review:
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
I guess you can use [the available template](https://github.com/pagopa/dx/blob/main/.github/workflows/js_code_review.yaml)?

Reply:
The template works only with turbo 1.x @lucacavallaro, we need to downgrade turbo in order to use the dx template?

Reply:
@lucacavallaro i dont have the permission on dx repo , unfortunately i cant open any pr

Reply:
Pr opened: https://github.com/pagopa/dx/pull/100

Reply:
updated Dx template and use it here!